spring/src/test/java/org/axonframework/spring/config/SpringAxonAutoConfigurerTest.java

```diff
@@ -362,7 +361,6 @@ void testHandlerDefinitionAndHandlerEnhancerBeansRegistered() {
                         MethodQueryMessageHandlerDefinition.class,
                         ReplayAwareMessageHandlerWrapper.class,
                         DeadlineMethodMessageHandlerDefinition.class,
-                        SecuredMessageHandlerDefinition.class,
                         MethodCreationPolicyDefinition.class,
                         MethodCreationPolicyDefinition.class,
                         MyHandlerEnhancerDefinition.class,
```

Comment:
Those handlers are always present in Axon framework for generic framework features like Deadlines. The Secured one is something additional.